master/internal/api_workspace.go

```diff
@@ -615,7 +616,49 @@ func (a *apiServer) UnpinWorkspace(
 }
 
 func (a *apiServer) ListRPsBoundToWorkspace(
-	ctx context.Context, request *apiv1.ListRPsBoundToWorkspaceRequest,
+	ctx context.Context, req *apiv1.ListRPsBoundToWorkspaceRequest,
 ) (*apiv1.ListRPsBoundToWorkspaceResponse, error) {
-	return &apiv1.ListRPsBoundToWorkspaceResponse{}, nil
+	curUser, _, err := grpcutil.GetUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+	err = workspace.AuthZProvider.Get().CanGetWorkspaceID(
```

Comment:
would this be the same functionality as `CanGetWorkspace`? I'd prefer using that instead.